client/http_curl.h

```diff
@@ -60,6 +60,8 @@ class HTTP_OP {
     char m_url[1024];  
     char m_curl_user_credentials[1024];
         // string needed for proxy username/password
+    char m_http_version[100];
```

Comment:
This information not need to be stored, so please remove these two member variables